decentraland/unity-renderer
# fix: white image stuck in onboarding

## What does this PR change?

Fixes #4629

To white image issue came from enabling a texture before it has been fetched. This PR reorganizes the call

The stuck situation came from changing realms at start, and generated race conditions with scenes not being unloaded properly. This PR takes you directly to the onboarding realm if the flag is on 

Also cleans old loading screen remanant code

## How to test the changes?

1. Go to zone (only place where flag is enabled) in incognito window. 
2. Create a user
3. Teleport out of the onboarding (you can use the genesis plaza button, or complete the onboarding)
4. Get to Genesis Plaza
5. Refresh
6. Check that the onboarding did not start again
7. With another already created account, check that you are not being taken to the onboarding

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
